perf/stability/setup_test.sh

```diff
@@ -17,17 +17,15 @@
 set -ex
 
 # shellcheck disable=SC2086
-WD=$(dirname $0)
-# shellcheck disable=SC2086
-WD=$(cd $WD; pwd)
+WD=$(cd $(dirname $0); pwd)
 
 function setup_test() {
   local DIRNAME="${1:?"test directory"}"
   local NAMESPACE="istio-stability-${NAMESPACE:-"$1"}"
   local HELM_ARGS="${2:-}"
 
   mkdir -p "${WD}/tmp"
-  local OUTFILE="${WD}/tmp/${DIRNAME}.yaml"
+  local OUTFILE="./perf/stability/tmp/${DIRNAME}.yaml"
```

Comment:
I think make it to be "./${WD}/tmp/${DIRNAME}.yaml" would be better than hard coded that path.

Reply:
Because I feel like the value of line 22 is weird, arbitrarily not used here

Reply:
output
```
+ WD=./perf/stability
++ cd ./perf/stability
++ pwd
+ WD=/work/perf/stability
```
Initially the path was correct, but after executing line 22, I thought it was wrong
```
WD=$(cd $WD; pwd)
```

Reply:
do you understand what's going on?

Reply:
Did you run the script inside container?

Reply:
No, documents are very vague, I don't know how to run `Stability Tests`

Reply:
I just got the information that I should do make stability on `tools/perf/stability`, but `tools/perf/stability` does not have a `Makefile`!